### PR TITLE
YAML to JSON translations

### LINKED ISSUE #132: [US] Books: MARC21 to JSON translations
Create the Marc21 to JSON translations based on the attached file created by Anne.Gentil-Beccot@cern.ch
[migration_conversion.xlsx](https://github.com/CERNDocumentServer/cds-dojson/files/2201213/migration_conversion.xlsx)

### PR DESCRIPTION
**Pull request by ioanungurean**
- [x] add a command to CLI

Closes #132 #134

TO BE MERGED AFTER #146

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -47,6 +47,7 @@
     'pytest-pep8>=1.0.6',
     'pytest>=2.8.0',
     'pycountry>=17.5.14',
+    'PyYAML>=3.13',
```
Comment (egabancho) [inline setup.py]: You probably you don't need this here since you added it already inside `install_requires`.

Comment (ioanungurean) [reply setup.py]: @egabancho you are right, I removed the `PyYAML` and `pycountry` it was added in both places.

**Comment by ludmilamarian** [comment]
@ioanungurean can you please rebase?

**Comment by ioanungurean** [comment]
@ludmilamarian I rebased, but should we wait for complete and proper data model from the library before we merge. WDYT?

**Comment by ludmilamarian** [comment]
@ioanungurean the data model has been merged already. I think we should work on top of what we have, and once we receive a new iteration on top of the data model, then add a new commit to it. In any case, the yaml to json will not change, right?

**Comment by ioanungurean** [comment]
@ludmilamarian yes the script will not change only the translated JSONs

**Pull request closed without merging**